src/services/role/index.js

```diff
@@ -40,11 +40,33 @@ module.exports = function setup() {
 
 	definePermissions(
 		'ADMIN_TOGGLE_STUDENT_VISIBILITY',
-		ROLES.TEACHER,
+		[ROLES.TEACHER],
 		PERMISSIONS.STUDENT_LIST,
 		// TODO: importent add this again, after school permission service can handle multiple permissions
 		// PERMISSIONS.STUDENT_EDIT,
 		// PERMISSIONS.STUDENT_CREATE,
 		// PERMISSIONS.STUDENT_DELETE,
 	);
+
+
+	//TODO: check if we need to adapt this function so it can check for certain values or conditions instead of only true
```

Comment:
Codacy found an issue: [This line has a length of 121. Maximum allowed is 120.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6051638)